src/common/src/aws/sdk/kotlin/crt/io/TlsCipherPreference.kt

```diff
@@ -11,6 +11,7 @@ package aws.sdk.kotlin.crt.io
  * At present, the ability to configure arbitrary orderings of TLS Ciphers is not allowed, and only a curated list of
  * vetted TlsCipherPreference's are exposed.
  */
+/* ktlint-disable enum-entry-name-case */
```

Comment:
**Comment**: I presume this is because, e.g., `…TLSv1…` would become `…TLSV1…` and that's hard to read? I suggest going with `…TLS_V1…` instead to avoid needing the rule override.

Reply:
I put the override on both of these because I didn't want to break anyone, but I should have questioned the assumption that they were public to begin with. They are not and thus they are fixed!